spec/models/database_spec.rb

```diff
@@ -0,0 +1,128 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe "Database" do
+  let(:models_todo) { ["ColumnPreference"].freeze }
```

Comment:
Cool, so this will need 23 more entries for now? Ah, no, once these migrations are actually applied then we need only 8 more in here.

Reply:
Well, should I just add the 6 (some models have two associations) now and still have the spec fail until you make the migrations or add them all now and make another PR with just the 7 troublesome models listed as todo?

Reply:
You can run the migrations locally which will alter the schema.rb. That needs to be part of the pull request. You then need to run `rails db:test:prepare` for your local test database to be updated with the new foreign keys. And the test should not detect missing keys there.

Production servers will still see that they are missing migrations and they will run then regardless of what's defined in the schema.rb.